src/controller/data_model/controller-clusters.matter

```diff
@@ -3554,6 +3554,113 @@ client cluster ActivatedCarbonFilterMonitoring = 114 {
   command ResetCondition(): DefaultSuccess = 0;
 }
 
+/** This cluster is used to configure a boolean sensor. */
+provisional client cluster BooleanSensorConfiguration = 128 {
+  enum SensitivityEnum : enum8 {
+    kHigh = 0;
+    kStandard = 1;
+    kLow = 2;
+  }
+
+  bitmap AlarmModeBitmap : bitmap8 {
+    kVisual = 0x1;
+    kAudible = 0x2;
+  }
+
+  bitmap Feature : bitmap32 {
+    kVisual = 0x1;
+    kAudible = 0x2;
+    kAlarmSupress = 0x4;
+    kSensitivityLevel = 0x8;
+  }
+
+  info event AlarmsStateChanged = 0 {
+    AlarmModeBitmap alarmsActive = 0;
+    optional AlarmModeBitmap alarmsSuppressed = 0;
+  }
+
+  info event SensorFault = 1 {
+  }
+
+  attribute optional SensitivityEnum sensitivityLevel = 0;
+  readonly attribute optional AlarmModeBitmap alarmsActive = 1;
+  readonly attribute optional AlarmModeBitmap alarmsSuppressed = 2;
+  attribute optional AlarmModeBitmap alarmsEnabled = 3;
+  readonly attribute command_id generatedCommandList[] = 65528;
+  readonly attribute command_id acceptedCommandList[] = 65529;
+  readonly attribute event_id eventList[] = 65530;
+  readonly attribute attrib_id attributeList[] = 65531;
+  readonly attribute bitmap32 featureMap = 65532;
+  readonly attribute int16u clusterRevision = 65533;
+
+  request struct SupressRequestRequest {
```

Comment:
Stupid question, is the "RequestRequest" intentional?

Reply:
Because the command in the spec is named "SuppressRequest".

Reply:
So the latter "Request" is just some added naming? We can certainly discuss renaming this if it makes more sense (also on the XML side)

Reply:
Yes, commands have a request and a response. That is what the codeine does. Well, this one doesn't have an explicit response. Improving the name would be welcomed from my side.

Reply:
The IDL names are not used for anything in particular.  Other things that define structs to represent commands name them differently.